plans/utils/deploy_agents.pp

```diff
@@ -0,0 +1,31 @@
+# @summary Destroy a pecdm provisioned PE cluster
```

Comment:
I know picking at comments is just fussy

Reply:
I've fixed them all in local copy that I've not yet pushed.